### ISSUE TITLE
Video player component

### ISSUE THREAD

**Issue by kttalley** [surajondev/UI-UX-Design#115]
Hey there, @surajondev. I’m new to contributing to open source projects and just found this lovely repo. I see there aren’t any video related UI components, such as custom media players and the like. Would you be open to me contributing a custom video player component I’ve created? It does leverage some JS— should all the components contributed here be HTML/CSS only? Thanks for the clarification. Hope I can be of assistance.

**Comment by surajondev** [comment]
It's a nice idea. You can use JS too if it requires. I am assigning you to work on this.

**Comment by kttalley** [comment]
Thanks so much @surajondev. I've attempted to commit my video player component to your main repository here: https://github.com/surajondev/UI-UX-Design/pull/119/commits/3801fb741f5a9ea174a0328a4cea3910d79c5c0b

Apologies if I did not correctly associate this assigned issue with my submitted merge request. Of course let me know if there are any modifications or clarification I can provide. Thanks again for the assistance and opportunity to learn here.
